test/integration/scheduler/podsready/scheduler_test.go

```diff
@@ -650,7 +641,7 @@ var _ = ginkgo.Describe("SchedulerWithWaitForPodsReadyNonblockingMode", func() {
 				Obj()
 			gomega.Expect(k8sClient.Create(ctx, testCQ)).Should(gomega.Succeed())
 			defer func() {
-				gomega.Expect(util.DeleteObject(ctx, k8sClient, testCQ)).Should(gomega.Succeed())
+				util.ExpectObjectToBeDeleted(ctx, k8sClient, testCQ, true)
```

Comment:
I don't see how this change is related. Please move to a dedicated PR if not needed here.

Reply:
same